selfdrive/car/mazda/values.py

```diff
@@ -8,18 +8,20 @@
 # Steer torque limits
 
 class CarControllerParams:
-  STEER_MAX = 600                 # max_steer 2048
-  STEER_STEP = 1                  # how often we update the steer cmd
+  STEER_MAX = 800                # theoretical max_steer 2047
   STEER_DELTA_UP = 10             # torque increase per refresh
-  STEER_DELTA_DOWN = 20           # torque decrease per refresh
+  STEER_DELTA_DOWN = 25           # torque decrease per refresh
   STEER_DRIVER_ALLOWANCE = 15     # allowed driver torque before start limiting
   STEER_DRIVER_MULTIPLIER = 1     # weight driver torque
   STEER_DRIVER_FACTOR = 1         # from dbc
+  STEER_ERROR_MAX = 350           # max delta between torque cmd and torque motor
 
 class CAR:
-  CX5 = "Mazda CX-5 2017"
-  CX9 = "Mazda CX-9 2017"
-  Mazda3 = "Mazda3 2017"
+  CX5 = "Mazda CX-5"
```

Comment:
Can we make the car names match the other brands? Keys and values should be all caps, so `MAZDA3 = MAZDA 3`. Would be nice if you can also find a year or generation name for the older cars.

Reply:
I did  change the name. I think I'm going to hold on giving a generation name for now until we see a new generation upstreams. I already refer to these cars as "Gen1" Mazda ADAS system which include most Mazda cars so far. the new Mazda3 and CX-30 have a new system, we will call that Gen2 when it is supported.